tests/client-dbus/src/stratisd_client_dbus/_data.py

```diff
@@ -62,6 +62,20 @@
 <arg name="return_code" type="q" direction="out"/>
 <arg name="return_string" type="s" direction="out"/>
 </method>
+<method name="SetKey">
```

Comment:
This chunk of introspection data found its way into the wrong Manager interface.